crates/rpc/src/lib.rs

```diff
@@ -14,8 +15,12 @@ pub mod types;
 pub mod utils;
 
 /// Start the Beacon API server.
-pub async fn start_server(network_spec: Arc<NetworkSpec>, server_config: ServerConfig) {
-    let routes = get_routes(network_spec).recover(handle_rejection);
+pub async fn start_server(
+    network_spec: Arc<NetworkSpec>,
+    server_config: ServerConfig,
+    db: Arc<ReamDB>,
```

Comment:
```suggestion
    db: ReamDB,
```

we don't need this arc